pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1539,6 +1539,8 @@ type GatewaySpec struct {
 	Service *RGWServiceSpec `json:"service,omitempty"`
 
 	// Whether host networking is enabled for the rgw daemon. If not set, the network settings from the cluster CR will be applied.
+	// If host networking is enabled or disabled on a running cluster, then the operator will automatically failover all the mons to
+	// apply the new network settings.
```

Comment:
This is an unnecessary note for the CephObjectStore.Gateway spec.

Reply:
Right, this comment was not meant to be here. Moved it to  cephCluster spec.